src/chainlit/frontend/src/theme.ts

```diff
@@ -82,26 +83,26 @@ const lightTheme = createTheme({
     success,
     error,
     background: {
-      default: '#FAFAFA',
-      paperVariant: '#EEEEEE',
-      paper: '#FFF'
+      default: grey[50],
+      paperVariant: grey[200],
```

Comment:
we can get rid of paperVariant et use the grey palette directly

Reply:
✅